examples/models/metadata/graph_metadata.ipynb

```diff
@@ -21,18 +21,34 @@
   },
```

Comment:
<p>Would be useful to have the make commands to build the images</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/2005/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
There are Makefiles in `examples/models/metadata/models/...`, for example this one 

https://github.com/SeldonIO/seldon-core/blob/e6abbce6700d272024ef3980a54f3345adddc00d/examples/models/metadata/models/generic-node/Makefile#L1-L29

Reply:
Oh what I meant is that it would be good to have the command on the notebook, as right now it doesn't specify how to build it

Reply:
Ah yes... I was following what recently had in examples that the model was only only showed but not build. 
This is because build would have to follow with `kind load` or similar and that would be very platform specific. 

Shall we assume that example notebook is run in `kind` cluster?

Reply:
Adding note referencing the Makefile.